cargo-dist/src/sign/mod.rs

```diff
@@ -34,6 +45,18 @@ impl Signing {
                 signer.sign(file)?;
             }
         }
+        if let Some(signer) = &self.macos {
+            // TODO: restructure, this is just to keep Windows
+            // from flagging dead code
+            #[cfg(unix)]
+            let is_executable = file.metadata()?.permissions().mode() & 0o111 != 0;
```

Comment:
Are there non-executable things macos can/should sign?

Reply:
I'll leave a comment. At the moment, no. In the future if we end up distributing `.app` bundles or other bundles, we'll want to sign those.

The `.pkg` signing is going to be handled by `pkgbuild`/`productbuild`, so we don't need to handle that here.